### ISSUE TITLE
System.Data.SQLite version error

### ISSUE THREAD

**Issue by Ash-IPRANOps** [judero01col/GMap.NET#145]
Hello,
  Can you please update the project to use the latest System,.Data.SQLite v1.0.115?
 I can't GMAP anymore because I can't find  v1.0.113 anywhere to downgrade the one I have. please :(

![image](https://user-images.githubusercontent.com/43791859/136279880-3e20a549-4bec-4668-b0dc-636aee6c76de.png)

**Comment by IGP67** [comment]
We have same problem, update by resolution would be helpful.

**Comment by krnrpl** [comment]
This is usually fixed by an bindingRedirect in `app.config` no? At least that's what I use (and is automatically done so when installing the nuget package).
```
<dependentAssembly>
	<assemblyIdentity name="System.Data.SQLite" publicKeyToken="db937bc2d44ff139" culture="neutral" />
	<bindingRedirect oldVersion="0.0.0.0-1.0.115.0" newVersion="1.0.115.0" />
</dependentAssembly>
```

**Comment by Ash-IPRANOps** [comment]
I was trying to avoid using app.config. but yes that should make it work too. Thanks.

**Comment by energywave** [comment]
I'm also having this problem and no, the app.config is not a solution as I'm creating a dll file that is loaded by an old VB6 exe through ActiveX and it seems that app.config is not working (I also have the bindingredirect created by nuget).
I'm going to recompile this project and include it in the solution... But a permanent solution would be great.

**Comment by judero01col** [comment]
please test if version 2.1.4 fixes your problem.

**Comment by energywave** [comment]
Sorry, I missed the notify about this comment but I then made a PR with framework 4.8 and updated dependencies that you merged and then reverted. I then asked why did you revert my PR but got no reply.
However I'm sorry but I don't have time just now to test this release as my solution is now with my own version of your project from where I made the PR. At the first available time I'll test it, thank you.
